src/modules/AspectRatio/Module.js

```diff
@@ -0,0 +1,59 @@
+/*
+* Crops an Image on the basic of the aspect ratio
+*/
+module.exports = function AspectRatio(options, UI) {
+
+  var defaults = require('./../../util/getDefaults.js')(require('./info.json'));
+  var output;
+
+  function draw(input, callback, progressObj) {
+
+    var step = this;
+    var startingX = Number(options.startingX || defaults.startingX);
+    var startingY = Number(options.startingY || defaults.startingY);
+    var aspectRatio = (options.aspectRatio || defaults.aspectRatio).split(':');
+    var widthRatio = Number(aspectRatio[0]);
+    var heightRatio = Number(aspectRatio[1]);
+
+    progressObj.stop(true);
+    progressObj.overrideFlag = true;
+
+    var getPixels = require('get-pixels');
```

Comment:
We recently added experimental GIF support. It would be better if you use `lodash` to `cloneDeep` the pixels instead of reading it from the source.
It would be faster as well as would work with GIFs. Please refer to the FlipImage module or Crop module.
Thanks